src/vcpkg/export.cpp

```diff
@@ -263,6 +262,12 @@ namespace vcpkg::Export
             fs.create_directories(destination.parent_path(), IgnoreErrors{});
             fs.copy_file(source, destination, CopyOptions::overwrite_existing, VCPKG_LINE_INFO);
         }
+
+        // Copying exe (this is not relative to root)
+        Path vcpkg_exe = get_exe_path_of_current_process();
+        auto destination = raw_exported_dir_path / "vcpkg.exe";
```

Comment:
@ras0219-msft is this right? (the .exe)

Reply:
I have reverted the changes on this file as recommended by @ras0219-msft